tests/release/pipelines/push_binaries_to_dev_portal.go

```diff
@@ -0,0 +1,292 @@
+)
+
+var devportalComponent *appservice.Component
+var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for push-binaries-to-dev-portal pipeline", Label("release-pipelines", "push-binaries-to-dev-portal"), func() {
+	defer GinkgoRecover()
+
+	var devWorkspace = utils.GetEnv(constants.RELEASE_DEV_WORKSPACE_ENV, constants.DevReleaseTeam)
+	var managedWorkspace = utils.GetEnv(constants.RELEASE_MANAGED_WORKSPACE_ENV, constants.ManagedReleaseTeam)
+
+	var devNamespace = devWorkspace + "-tenant"
+	var managedNamespace = managedWorkspace + "-tenant"
+
+	var err error
+	var devFw *framework.Framework
+	var managedFw *framework.Framework
+	var devportalApplicationName = "devportal-app-" + util.GenerateRandomString(4)
+	var devportalComponentName = "devportal-comp-" + util.GenerateRandomString(4)
+	var devportalReleasePlanName = "devportal-rp-" + util.GenerateRandomString(4)
+	var devportalReleasePlanAdmissionName = "devportal-rpa-" + util.GenerateRandomString(4)
+	var devportalEnterpriseContractPolicyName = "devportal-policy-" + util.GenerateRandomString(4)
+
+	var snapshot *appservice.Snapshot
+	var releaseCR *releaseapi.Release
+	var buildPR *tektonv1.PipelineRun
+
+	AfterEach(framework.ReportFailure(&devFw))
+
+	Describe("Push-to-dev-portal happy path", Label("pushToDevPortal"), func() {
+		BeforeAll(func() {
+			devFw = releasecommon.NewFramework(devWorkspace)
+			managedFw = releasecommon.NewFramework(managedWorkspace)
+			// Create a ticker that ticks every 3 minutes
+			ticker := time.NewTicker(3 * time.Minute)
+			// Schedule the stop of the ticker after 15 minutes
+			time.AfterFunc(15*time.Minute, func() {
+				ticker.Stop()
+				fmt.Println("Stopped executing every 3 minutes.")
+			})
+			// Run a goroutine to handle the ticker ticks
+			go func() {
+				for range ticker.C {
+					devFw = releasecommon.NewFramework(devWorkspace)
+					managedFw = releasecommon.NewFramework(managedWorkspace)
+				}
+			}()
+
+			managedNamespace = managedFw.UserNamespace
+
+			// Linking the build secret to the pipeline service account in dev namespace.
+			err = devFw.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(devNamespace, releasecommon.HacbsReleaseTestsTokenSecret, constants.DefaultPipelineServiceAccount, true)
+			Expect(err).ToNot(HaveOccurred())
+
+			keyExodusProd := os.Getenv(constants.EXODUS_PROD_KEY_ENV)
+			Expect(keyExodusProd).ToNot(BeEmpty())
+
+			certExodusProd := os.Getenv(constants.EXODUS_PROD_CERT_ENV)
+			Expect(certExodusProd).ToNot(BeEmpty())
+
+			// Creating k8s secret to access Exodus Non product based on base64 decoded of key and cert
+			exodusKeyDecoded, err := base64.StdEncoding.DecodeString(string(keyExodusProd))
+			Expect(err).ToNot(HaveOccurred())
+
+			exodusCertDecoded, err := base64.StdEncoding.DecodeString(string(certExodusProd))
+			Expect(err).ToNot(HaveOccurred())
+
+			exSecret, err := managedFw.AsKubeAdmin.CommonController.GetSecret(managedNamespace, releasecommon.ExodusStageSecretName)
```

Comment:
can you call this exodusSecret instead of exSecret? I know you are just trying to abbreviate but I find it a little confusing here

Reply:
Updated. Thanks